transform-plugins/docs/Hasher-transform.md

```diff
@@ -3,11 +3,14 @@
 
 Description
 -----------
-Hashes fields using a digest algorithm such as ``MD2``, ``MD5``, ``SHA1``, ``SHA256``, ``SHA384``, or ``SHA512``.
-
+Hashes fields using a digest algorithm such as ``MD2``, ``MD5``, ``SHA1``, ``SHA256``, ``SHA384``, or ``SHA512``.<br>
 
 Configuration
 -------------
 **fields:** Specifies the fields to be hashed.
 
 **hash:** Specifies the hashing algorithm.
+
+Prerequisites
+-------------
+The **fields** to be hashed must be of type `string` and `non-nullable`.
```

Comment:
Put this as a sentence in the fields description instead of in a separate section.

Reply:
@albertshau Thanks for reviewing, made the change as per suggested.